storage/database/db_manager.go

```diff
@@ -232,19 +234,21 @@ var dbDirs = [databaseEntryTypeSize]string{
 	"statetrie_", // not used
 	"txlookup",
 	"bridgeservice",
+	"stakinginfo",
 }
 
 // Sum of dbConfigRatio should be 100.
 // Otherwise, logger.Crit will be called at checkDBEntryConfigRatio.
 var dbConfigRatio = [databaseEntryTypeSize]int{
 	3,  // MiscDB
 	6,  // headerDB
-	16, // BodyDB
-	16, // ReceiptsDB
+	15, // BodyDB
+	15, // ReceiptsDB
 	19, // StateTrieDB
 	19, // StateTrieMigrationDB
-	17, // TXLookUpEntryDB
+	16, // TXLookUpEntryDB
 	4,  // bridgeServiceDB
+	3,  // StakingInfoDB
```

Comment:
Why don't you use `MiscDB` like istanbul snapshot?

Reply:
I agree if staking info is stored only for migration.
However, it is stored everyday. I think making a staking info db is better than combining it to miscdb.

Reply:
`GovernanceState` is also stored in `MiscDB`.
I agrees that it is good to store `stakingInfo` and `governanceState` in separate DB.
But, if we add new DB like this, the DB cache and FD limit will be reduced.
I am worried about that.
We need to check making more DB is ok or not.